# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/base/gcc/gcc2/gcc33_support.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./gcc/ch/loop.c.orig	2003-08-03 12:26:04.000000000 -0400
+++ ./gcc/ch/loop.c	2003-08-03 12:28:30.000000000 -0400
@@ -318,8 +318,8 @@
 (associated with all iterators).
 
  NOTE: Z.200, section 10.1 says that a block is ...
-       "the actions statement list in a do action, including any
-       loop counter and while control".  This means that an exp-
+       the actions statement list in a do action, including any
+       loop counter and while control.  This means that an exp-
        ression in a WHILE control can include references to the
        loop counters created for the loop''s exclusive use.  
        Example:
